source/features/clean-conversation-headers.css

```diff
@@ -46,3 +46,8 @@
 .gh-header.issue .rgh-clean-conversation-headers relative-time {
 	display: none;
 }
+
+/* Fixes vertical alignment of base branch dropdown */
+.rgh-clean-conversation-headers .commit-ref-dropdown {
+	margin-top: -2px !important;
+}
```

Comment:
This could technically be part of `github-bugs.css` since afaict it's not caused by this feature.

Reply:
It should have an issue number though